.storybook/themes/news.scss

```diff
@@ -879,6 +879,30 @@
 					font-size: var(--global-font-size-13),
 					font-weight: var(--global-font-weight-7),
 				),
+				large-manual-promo: (
```

Comment:
all of this should also be in feature pack

Reply:
updated in the feature-pack